docker/dev/find_fastest_resources.sh

```diff
@@ -24,9 +24,18 @@
 # function find_fastest_maven_repo()  echos fastest maven repo
 # function find_fastest_go_proxy()    echos fastest go proxy
 # function find_fastest_docker_url()  echos fastest docker download url
-# function find_fastest_docker_mirror echos fastest docker mirror url
+# function find_fastest_docker_mirror() echos fastest docker mirror url
+# function find_fastest_pip_mirror()  echos fastest pip mirror config
 #
 
+apt-get update
+
+if ! which ping >/dev/null; then
```

Comment:
Because this script file is in `/docker/` and the one place to call it is the Dockerfile, I don't think we need the `if` control flow.